cmake/IncludeAbseil.cmake

```diff
@@ -30,6 +30,7 @@ endif ()
 add_subdirectory(${ABSEIL_ROOT_DIR} third_party/abseil EXCLUDE_FROM_ALL)
 set(ABSEIL_LIBRARIES abseil)
 set(ABSEIL_INCLUDE_DIRS ${ABSEIL_ROOT_DIR})
+set(ABSL_STD_CXX_FLAG "-std=c++11" CACHE STRING "c++ std flag (default: c++11)")
 
 if(MSVC)
 target_compile_definitions(absl_base PUBLIC -DNOMINMAX -DWIN32_LEAN_AND_MEAN)
```

Comment:
Can you indent that by 4 please?  Not your fault, I just noticed it.

Reply:
Done.